src/cryptoadvance/specter/util/wallet_importer.py

```diff
@@ -187,7 +187,7 @@ def parse_wallet_data_import(cls, wallet_data):
 
         # Electrum singlesig
         elif "keystore" in wallet_data:
-            wallet_name = wallet_data["keystore"]["label"]
+            wallet_name = wallet_data["keystore"].get("label", "Electrum Wallet")
```

Comment:
It's really difficult to read the code without having the json as example somewhere noted. Recently in the chat we had also import issues and i made the point that the formats are not standardized. Can you attach examples as comment right in the code here?
Maybe at best in the method-documentation?